style.css

```diff
@@ -144,11 +146,35 @@
     display: flex;
     justify-content: space-evenly;
     align-items: center;
+    margin-top: 1rem;
+    padding: 1rem;
   }
   .code-container {
     margin-top: 1rem;
     display: flex;
     justify-content: center;
   }
+
+  .status{
+      width: 1rem;
+      height: 1rem;
+      position: absolute;
+      background-color: red;
+      border-radius: 50%;
+      margin-top: -1.4em;
+      margin-left: 3.5rem;
+  }
+   .status-online{
+      border: 2px solid green;
+      background-color: green;
+
+  }
+  .status-online-offline{
+      border: 2px solid rgb(15, 17, 15);
+      background-color: rgb(200, 214, 200);
```

Comment:
Same here, the use of color variables would be beneficial.

Reply:
thanks for the review.